fern/docs/pages/overview/define-your-api/ferndef/export-openapi.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: Fern | Export from Fern Definition to OpenAPI
+description: Export your Fern Definition files to OpenAPI using Fern's OpenAPI generator.
+---
+
+To prevent lock-in to the Fern Definition format, we provide a generator that will export your Fern Def files to OpenAPI 3.1. This lets you switch to using OpenAPI at any time, or use your API definition with OpenAPI tools. For more information, see our [OpenAPI generator](/fern-sd-ks/other-generators/spec/openapi-generator). 
```

Comment:
```suggestion
To prevent lock-in to the Fern Definition format, we provide a generator that will export your Fern Def files to OpenAPI 3.1. This lets you switch to using OpenAPI at any time, or use your API definition with OpenAPI tools. For more information, see our [OpenAPI generator](/fern-sd-ks/other-generators/spec/openapi). 
```